tldw_Server_API/app/core/DB_Management/media_db/runtime/defaults.py

```
@@ -37,47 +38,67 @@
-        raise RuntimeError(
-            "PostgreSQL content backend is required but was not initialized. "
-            "Check TLDW_CONTENT_DB_BACKEND configuration."
-        )
-    return backend
+    with _runtime_state_context():
+        backend = ensure_content_backend_loaded()
+        if postgres_content_mode and backend is None:
+            raise RuntimeError(
+                "PostgreSQL content backend is required but was not initialized. "
+                "Check TLDW_CONTENT_DB_BACKEND configuration."
+            )
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)